src/map/script.c

```diff
@@ -21540,6 +21572,8 @@ BUILDIN(bindatcmd)
 		atcommand->binding[i]->group_lv = group_lv;
 		atcommand->binding[i]->group_lv_char = group_lv_char;
 		atcommand->binding[i]->log = log;
+		CREATE(atcommand->binding[i]->at_groups, char, db_size(pcg->db));
```

Comment:
`db_size(pcg->db)` doesn't seem enough. Group IDs aren't sequential, but arbitrarily assigned. By default we ship with a group having ID = 99, but we have less than 99 groups.